requester/report.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/benchttp/runner/config"
 )
 
+const (
+	defaultRecordsCap = 128
```

Comment:
Why 128 specifically ? Does it get reallocated if it exceeds?

Reply:
Because why not 😛 The cap algorithm is very simple: it reallocates the double when reached. So for 1000 requests, it'll be reallocated 3 times.
These amount of space are not gonna be an issue, so I think we can go for `1000` by default.